samples/kctf-conf/base/Makefile

```diff
@@ -279,8 +279,8 @@ ${CLUSTER_GEN}/remote-healthcheck-image: ${CLUSTER_GEN}/healthcheck-image-tagged
 	@  echo 'error: cluster config not loaded. Run kctf-config-create or kctf-config-load'
 	@  exit 1
 	@fi
-	kubectl config use-context "kctf_${PROJECT}_${ZONE}_${CLUSTER_NAME}"
-	mkdir -p ${CLUSTER_GEN}
+	kubectl config use-context "kctf_${PROJECT}_${ZONE}_${CLUSTER_NAME}" >&2
+	mkdir -p ${CLUSTER_GEN} >&2
```

Comment:
can we just do > /dev/null instead? The stdout of those commands is not interesting anyway and writing to stderr should IMO only be done for errors and warnings. I.e. I had a friend who would color stderr output in red which would make this weird.